subjects/atm-management-system/audit/README.md

```diff
@@ -6,73 +6,73 @@
+##### Referring to the user Alice, create an account but with account number `"320421"` and type of account `"fixed01"` with the rest of the information the same as the last account. Then select `"Check accounts"` and choose the account you just created.
 
-###### Did the application displayed the account information and the gain of $40.05 of interest on 10/10/2013?
+###### Did the application display the account information and the gain of $40.05 of interest on 10/10/2013?
 
-##### Resorting to the user Alice create again an account but with account number `"3214"` and type of account `"fixed02"` with the rest of the information as in the last account. Then select `"Check accounts"` and choose the account you just created.
+##### Referring to the user Alice, create an account but with account number `"3214"` and type of account `"fixed02"` with the rest of the information the same as the last account. Then select `"Check accounts"` and choose the account you just created.
 
-###### Did the application displayed the account information and the gain of $100.12 of interest on 10/10/2014?
+###### Did the application display the account information and the gain of $100.12 of interest on 10/10/2014?
 
-##### Resorting to the user Alice create again an account but with account number `"3212"` and type of account `"fixed03"` with the rest of the information as in the last account. Then select `"Check accounts"` and choose the account you just created.
+##### Referring to the user Alice, create an account but with account number `"3212"` and type of account `"fixed03"` with the rest of the information the same as the last account. Then select `"Check accounts"` and choose the account you just created.
 
-###### Did the application displayed the account information and the gain of $240.29 of interest on 10/10/2015?
+###### Did the application display the account information and the gain of $240.29 of interest on 10/10/2015?
 
-##### Resorting to the user Alice select the option `"Make transaction"`. Then choose the account with the id `"3212"`
+##### Referring to the user Alice, select the option `"Make transaction"`. Then choose the account with the id `"3212"`
 
 ###### Are you able to choose between withdrawing or depositing?
 
-##### Resorting to the user Alice select the option `"Make transaction"`, choose the account with the id `"3212"`. Then try to withdraw money.
+##### Referring to the user Alice select the option `"Make transaction"`, choose the account with the id `"3212"`. Then try to withdraw money.
 
 ###### Are you able to withdraw money?
 
 ###### And if so, was the withdraw updated in the file `"records.txt"`?
 
-###### Is it not possible to withdraw an amount superior to your available balance?
+###### Is it possible to withdraw an amount superior to your available balance?
 
```

Comment:
If you remove the "not" it will change the sentence.
in order to pass the questions the answer for it should be always "yes", otherwise it will fail the audit.
and if you say  yes to this question you are saying that it is possible to withdraw an amount superior to your available balance which should not be possible!
We can do it like this:

###### Can you confirm that it is not possible to withdraw more than your available balance?

Now if you say yes it is correct and it will pass the audit

Reply:
We should maybe change the sentence to: `Does the system forbid to withdraw an amount superior to your available balance`?